src/main/java/graphql/GraphQLConfiguration.java

```diff
@@ -0,0 +1,150 @@
+package graphql;
+
+import graphql.parser.ParserOptions;
+
+/**
+ * This allows you to control specific aspects of the GraphQL system
+ * including some JVM wide settings and some per execution settings
+ * as well as experimental ones
+ */
+public class GraphQLConfiguration {
```

Comment:
Other name suggestion `GraphQlSettings` and hence the name of `GraphQl.configuration()` would change